.clang-format

```diff
@@ -22,3 +22,33 @@ PointerAlignment: Left
 Standard: c++14
 TabWidth: 2
 UseTab: Never
+IncludeBlocks: Regroup
+IncludeCategories:
+  - Regex:           '^<pcl/'
+    Priority:        1
+  - Regex:           '^<gtest/'
+    Priority:        2
+  - Regex:           '^<boost/'
+    Priority:        3
+  - Regex:           '^<(unsupported/)?Eigen/'
+    Priority:        4
+  - Regex:           '^<flann/'
+    Priority:        5
+  - Regex:           '^<(Q|q)\w*/'
+    Priority:        6
+  - Regex:           '^<vtk/'
+    Priority:        7
+  - Regex:           '^<librealsense2/'
+    Priority:        8
+  - Regex:           '^<(ros|message_filters)/'
+    Priority:        9
+  - Regex:           '^<opencv(2)?/'
+    Priority:        10
+  - Regex:           '^<tide/'
+    Priority:        11
+  - Regex:           '^<thrust/'
+    Priority:        12
+  - Regex:           '^<(OpenGL|(GL(UT)?)/'
+    Priority:        13
+  - Regex:           '.*'
+    Priority:        14
```

Comment:
What about the includes using quotes: `#include "stuff"`?

We have quite some of them: [grep.txt](https://github.com/PointCloudLibrary/pcl/files/4466196/grep.txt)

Reply:
As the convention in PCL is always to have path relative to the root (`<pcl/...`) and not relative  the current dir, all occurrences of `#include ""` should be converted to `#include <>` during applying clang-format. See [here](https://gcc.gnu.org/onlinedocs/gcc-2.95.3/cpp_1.html#SEC6)